packages/datagateway-download/src/downloadApi.ts

```diff
@@ -98,23 +95,26 @@ export const submitCart: (
       params
     )
     .then((response) => {
-      log.debug(response);
-
       // Get the downloadId that was returned from the IDS server.
+      console.log('response', response);
       const downloadId = response.data['downloadId'];
       return downloadId;
-    })
-    .catch((error) => {
-      handleICATError(error);
-      return -1;
     });
 };
 
+/**
+ * Settings for {@link fetchDownloads}
+ */
+export interface FetchDownloadsSettings {
+  facilityName: string;
+  downloadApiUrl: string;
+}
```

Comment:
It seems like other functions in this file use this same type for the settings - therefore could this be renamed to be more generic and used for other functions like `getDownload`, `getDownloadStatusType` etc

Reply:
I actually tried to do that, but other functions take in settings with slightly different shapes, (e.g. `getDataFileCount` takes `{ apiUrl: string }`). Although we can use the `Pick` util type to pick the keys we want from the settings file.